applications/source/rabbit-source/README.adoc

```diff
@@ -0,0 +1,103 @@
+//tag::ref-doc[]
+= RabbitMQ Source
+
+The "rabbit" source enables receiving messages from RabbitMQ.
+
+The queue(s) must exist before the stream is deployed; they are not created automatically.
+You can easily create a Queue using the RabbitMQ web UI.
+
+== Input
+
+N/A
+
+== Output
+
+=== Headers
+
+* `content-type: text/plain`
+
+=== Payload
+
+* `String`
+
+=== Headers
+
+* `content-type: application/octet-stream`
+
+=== Payload
+
+* `byte[]`
+
+=== Headers
+
+* `content-type: application/x-java-serialized-object`
```

Comment:
I think this is wrong or irrelevant any more.
Probably by default it is an `application/json`

Reply:
Removed the output headers section. For the test the output content type is text/plain, not sure if I should do something to support other content types or if it is done under the hood.